src/Uno.Sdk/ReadMe.md

```diff
@@ -305,14 +305,14 @@ The Uno.Sdk powers the Uno Platform Single Project, including the ability to imp
   },
   {
     "group": "Maui",
-    "version": "8.0.100",
+    "version": "9.0.60",
     "packages": [
       "Microsoft.Maui.Controls",
       "Microsoft.Maui.Controls.Compatibility",
       "Microsoft.Maui.Graphics"
     ],
     "versionOverride": {
-      "net9.0": "9.0.100"
+      "net10.0": "9.0.60"
```

Comment:
Same for Maui, versionOverride was bumped in packages.json but not in the readme. 

@jeromelaban can we add a CI validation to make sure ReadMe content and packages.json always match?

<img width="465" height="224" alt="image" src="https://github.com/user-attachments/assets/db8e8f00-3be0-461d-85c3-aae096314caa" />

Reply:
There's no need, it will update itself on the next run